gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -685,8 +942,25 @@ properties:
       - Unknown
       - Not Reported
       - Not Allowed To Collect
+      - Stage IC1
+      - Stage IC2
+      - Stage IC3
+      - Stage IIIA1
+      - Stage IIIAi
+      - Stage IIIAii
+      - Stage IIIA2
```

Comment:
Is there a difference between these?
* `Stage IIIA1` and `Stage IIIAi`
* `Stage IIIA2` and `Stage IIIAii`

I'm asking from a data normalization point of view.  If they are the same, can we pick one over the other?  From the other enums, it looks like arabic numerals are already used for this suffix.

Reply:
I spoke to Bill about this.  They are different things.